piker/brokers/kraken.py

```diff
@@ -106,6 +119,15 @@ class Pair(BaseModel):
     ordermin: float  # minimum order volume for pair
 
 
+class Trade(BaseModel):
+    """Trade class that helps parse and validate ownTrades stream"""
+    reqid: str # kraken order transaction id
```

Comment:
If you're going to use `pydantic` you should try to type these to make it more purposeful. They even support a time format now iirc though not sure it'll work in this case.

Reply:
Ok so we're not going to try and tackle this i guess?

If we end up moving to `cryptofeed` anyway this might all be in vain. so yeah not gonna expect this any more.